### PR TITLE
[Monotone Museum Korean] Update newly added results

### PR DESCRIPTION
**Pull request by jbblily**
Translated newly added chart result. Please take a look if it has any problem. Thank you.

### CODE REVIEW

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 1]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          당신이 어디에서 왔는지, 누구였는지. 고향이나 가족과의 추억이 허무 너머로 사라져간다. 이제 영원히, 기억할 수 없겠지.
+          ＞ 출신을 상실한다. (특성도 사용 불가)
+        - |-
+          【허무의 상흔】
+          사지에 격통이 일었다. 당신의 몸은 허무의 상흔이 뚜렷하게 남아, 사라지지 않고, 그 몸을 좀먹는다.
+          ＞ 【행동치】를 -5 할 것（하한 1）
+        - |-
+          【허무의 침식】
+          고동이 불길한 박자로 울린다. 육체가, 영혼이, 침식되어 사라지는 것 같은 끔찍한 감각. 설령 보지 않더라도, 착실하게 당신은 허무로 향하고 있다…
+        ＞ 애프터 플레이에서 기본박탈치를 +3할 것. (상한9). 대가를지불할 수 있는지는 박탈체크시에 확정할 것.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:460]: This line seems to be indented one level too shallow, resulting in a syntax error. Please fix it.
```suggestion
          ＞ 애프터 플레이에서 기본박탈치를 +3할 것. (상한9). 대가를지불할 수 있는지는 박탈체크시에 확정할 것.
```

**Review by ochaochaocha3 — CHANGES_REQUESTED** [review]

File: test/data/MonotoneMuseum_Korean.toml [hunk 2]
```diff
@@ -249,7 +249,7 @@ rands = [
 [[ test ]]
 game_system = "MonotoneMuseum:Korean"
 input = "DTS"
-output = "歪み表(海)(56) ＞ 【酒の海】\n海の水が酒になる。水を飲んだすべての者は酔っ払い理性を失う。その様はまるで暗黒期に滅亡した“怠惰の古都”を彷彿とさせる。"
+output = "일그러짐표(바다)(56) ＞ 【술의 바다】\n바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다."
```
Comment (ochaochaocha3) [inline test/data/MonotoneMuseum_Korean.toml:252]: The `"`s inside a string must be escaped, or please use `“...”` (fullwidth quotation marks) as in Japanese output.

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 3]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          당신은 [전투불능]이 된다.
+        - |-
+          【육체의 붕괴】
+          당신은 2D6점의 HP를 잃는다.
+        - |-
+          【방심】
+          당신은 배드스테이터스 [방심]을 받는다.
+        - |-
+          【중압】
+          당신은 배드스테이터스 [중압]을 받는다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:42]: Test for this value failed:

```
<"징조표(6) ＞ 【중압】\n" + "배드 스테이터스［중압］을 받는다."> expected but was
<"징조표(6) ＞ 【중압】\n" + "당신은 배드스테이터스 [중압]을 받는다.">
```

Probably, the test case is not updated.

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 4]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          올풀림 속에서 허무가 스며나와 힘 약한 자들을 끌고 가 소멸시킨다. 씬에 등장하고 있는 엑스트라는 벌레 한 마리 조차 남지 않고 소멸하고 두 번 다시 나타나지 않는다.
+        - |-
+          【계절변화】
+          계절이 돌변한다. 1D6을 굴려 1이면 봄, 2면 여름, 3이면 가을, 4면 겨울, 5면 플레이하는 현재의 계절, 6이면 GM의 임의로 하라. 급격한 계절의 변화는 자연히 농작물과 사람들의 건강을 상하게 한다.
+        - |-
+          【올풀림】
+          세계가 갈라져 올풀림이 출현한다. 올풀림에 닿은 자들은 허무에 삼켜져 돌아올 수 없게 된다.
+        - |-
+          【이형화】
+          가람이 낳은 절망과 공포가 배어 씬 내의 모든 엑스트라는 모종의 이형화를 받는다. 이것을 치료할 주술은 없다. 이형의 무리 (IZ P237) * 1D6과 전투시켜도 좋다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:115]: The trailing space should be removed.

```suggestion
          가람이 낳은 절망과 공포가 배어 씬 내의 모든 엑스트라는 모종의 이형화를 받는다. 이것을 치료할 주술은 없다. 이형의 무리 (IZ P237) * 1D6과 전투시켜도 좋다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 5]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          계절이 돌변한다. 1D6을 굴려 1이면 봄, 2면 여름, 3이면 가을, 4면 겨울, 5면 플레이하는 현재의 계절, 6이면 GM의 임의로 하라. 급격한 계절의 변화는 자연히 농작물과 사람들의 건강을 상하게 한다.
+        - |-
+          【올풀림】
+          세계가 갈라져 올풀림이 출현한다. 올풀림에 닿은 자들은 허무에 삼켜져 돌아올 수 없게 된다.
+        - |-
+          【이형화】
+          가람이 낳은 절망과 공포가 배어 씬 내의 모든 엑스트라는 모종의 이형화를 받는다. 이것을 치료할 주술은 없다. 이형의 무리 (IZ P237) * 1D6과 전투시켜도 좋다. 
+        - |-
+          【죽음진행】
+          사람들의 마음에 허무가 퍼져 불안과 절망이 가득 차오른다. 도망칠 수 없는 공포로부터 도망치기 위해 사람들은 스스로, 혹은 무의식적으로 죽음으로 향하는 행동을 시작한다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:118]: The trailing space should be removed.

```suggestion
          사람들의 마음에 허무가 퍼져 불안과 절망이 가득 차오른다. 도망칠 수 없는 공포로부터 도망치기 위해 사람들은 스스로, 혹은 무의식적으로 죽음으로 향하는 행동을 시작한다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 6]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          가람이 낳은 절망과 공포가 배어 씬 내의 모든 엑스트라는 모종의 이형화를 받는다. 이것을 치료할 주술은 없다. 이형의 무리 (IZ P237) * 1D6과 전투시켜도 좋다. 
+        - |-
+          【죽음진행】
+          사람들의 마음에 허무가 퍼져 불안과 절망이 가득 차오른다. 도망칠 수 없는 공포로부터 도망치기 위해 사람들은 스스로, 혹은 무의식적으로 죽음으로 향하는 행동을 시작한다. 
+        - |-
+          【시간가속】
+          시간이 가속한다. 씬 내에 존재하는 여러 무생물과 생물은 2d6 년의 세월이 지난 것 처럼 행동한다. 생물이라면 노화한다. 실 잣는 이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않는다.
+        - |-
+          【시간역류】
+          시간이 국지적으로 역류한다. 씬 내에 존재하는 여러 무생물과 생물은 2d6년 전으로 돌아간다. 생물이라면 젊어진다. 이 결과, 제조년/출생년도 보다 전으로 돌아갔을 경우에는 허무에 삼켜저 소멸한다. 실 잣는 이는 【재봉】난이도 8의 판정에 성공하면 이 영향을 받지 않는다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:124]: The trailing space should be removed.

```suggestion
          시간이 국지적으로 역류한다. 씬 내에 존재하는 여러 무생물과 생물은 2d6년 전으로 돌아간다. 생물이라면 젊어진다. 이 결과, 제조년/출생년도 보다 전으로 돌아갔을 경우에는 허무에 삼켜저 소멸한다. 실 잣는 이는 【재봉】난이도 8의 판정에 성공하면 이 영향을 받지 않는다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 7]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          일그러짐으로 인한 불안과 공포는 사람들을 자포자기하게 만든다. 노상강도가 빈발하고 관헌은 강압적이 되면서 마을은 혁명과 폭동의 소문에 휩싸이고 형벌은 잔인해진다.
+        - |-
+          【평온무사】
+          너무 섬뜩할 정도로 아무 일도 일어나지 않는다. 아무 일도 일어나지 않는다. 실 잣는 이는 등골이 서늘해지는 듯한 두려움을 느낀다.
+        - |-
+          【역병 만연】
+          고통 속에 목숨을 앗아가는 역병이 만연한다. 등장하고 있는 캐릭터는 【육체】 난이도 8의 판정을 실시해, 실패하면 사독 5를 받는다.
+        - |-
+          【이단 심문】
+          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:139]: The trailing space should be removed.
```suggestion
          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 8]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다. 
+        - |-
+          【왜곡 출현】
+          올풀림의 파편으로부터, 악독하게도 사람을 덮치는 것 밖에 모르는 일그러짐이 나타나, 공포를 조장하며 사람들을 습격한다. GM이 원한다면 병마(IZ P238)X2와 전투를 벌이게 해도 좋다. 다른 이형도 좋다.
+        - |-
+          【악몽 출현】
+          장면 내의 모든 사람들은 가공할 공포의 꿈을 꾼다. 심약한 자들은 이정표에 매달리거나, 이단자를 불태우지 않으면 이 꿈에서 벗어날 수 없다고 생각할 것이다.
+        - |-
+          【쥐의 연회】
+          쥐 떼가 출현해 곡물을 먹어치우고 역병을 퍼뜨린다. 어린이나 노인이라면 물어 죽일 것이다. 큰 쥐(MM P240)*1D6 마리와 전투를 벌이게 해도 된다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:148]: The trailing space should be removed.

```suggestion
          쥐 떼가 출현해 곡물을 먹어치우고 역병을 퍼뜨린다. 어린이나 노인이라면 물어 죽일 것이다. 큰 쥐(MM P240)*1D6 마리와 전투를 벌이게 해도 된다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 9]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          세계의 일그러짐이 보스캐릭터를 휘감아 그를 감싸고 허무에 인도한다. 세계에서 보스캐릭터는 소거되고 소멸한다. 엔딩페이즈로.
+        - |-
+          【자기 희생】
+          표를 굴린 PC의 파트너 NPC 하나가 사망한다. NPC는 GM이 결정할 것. 표를 굴린 PC의 HP와 MP를 완전히 회복한다.
+        - |-
+          【생명 탄생】
+          세계의 일그러짐의 영향일까, 연극의 무대에 생명이 가득차고, 명동하기 시작한다. 당신들은 대신 무언가의 생물의 내장 위에 서 있다. 등장한 캐릭터는 전원 중독5를 받는다.
+        - |-
+          【왜곡 확대】
+          씬에 등장해있는 실잣는이가 아닌 NPC 한 명이 칠흑의 흉수 (MM 240p)로 변신한다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:170]: The trailing space should be removed.
```suggestion
          씬에 등장해있는 실잣는이가 아닌 NPC 한 명이 칠흑의 흉수 (MM 240p)로 변신한다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 10]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          주변의 공간이 일그러지고, 파괴적인 에너지가 충만해진다. 다음에 행하는 대미지롤에 +5D6한다.
+        - |-
+          【변조】
+          왼쪽은 오른쪽으로, 빨강은 파랑으로, 위는 아래고, 일그러짐이 몸의 움직임을 방해한다. 등장하고있는 캐릭터는 전원 [낭패]를 받는다.
+        - |-
+          【공간 소실】
+          연극의 무대가 연기처럼 사라진다. 압도적인 상실감이 PC들을 덮친다. 이제 세계에 남아있는것은 당신들 뿐일지도 모른다. 등장해있는 캐릭터 전원 [방심]을 받는다.
+        - |-
+          【생명 소실】
+          다음 씬 이후 엑스트라는 일절 등장하지 않는다. 연극의 무대가 된 장소에서 동물이나 인간의 모습이 존재하지 않게 된다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:185]: The trailing space should be removed.
```suggestion
          다음 씬 이후 엑스트라는 일절 등장하지 않는다. 연극의 무대가 된 장소에서 동물이나 인간의 모습이 존재하지 않게 된다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 11]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+        items:
+        - |-
+          【신념 상실】
+          당신의 정신이 당신의 과거를 부정한다. 당신은 [출신]과 그에 해당하는 기억을 그 씬 동안 상실한다. 특성으로 인한 효과는 잃지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【졸도】
+          더 이상 당신은 이탈의 부담을 버티는 것이 불가능하다. 그것이 세계에 거스른 대가이다. 당신은 [전투불능]이 된다. 이미 [전투불능]일 경우 [사망]한다.
+        - |-
+          【육체 붕괴】
+          유리가 깨지는 소리가 나며, 당신도 깨져간다. 당신은 2D6+캐릭터레벨 점의 【HP】를 잃는다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:261]: The trailing space should be removed.
```suggestion
          유리가 깨지는 소리가 나며, 당신도 깨져간다. 당신은 2D6+캐릭터레벨 점의 【HP】를 잃는다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 12]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          당신의 몸에서 색이 사라져 탁한 백색에 가까워진다. 옷이나 화장으로 숨기는 것은 가능하나 들키면 이단심문은 피할 수 없을테지. 당신의 박탈치를 +1d6한다 (이 효과로는 징조표가 굴러가지 않는다)
+        - |-
+          【이유 상실】
+          당신의 정신이 당신의 현재를 부정한다. 당신은 [경험]과 그것에 관한 기억을 그 씬 동안 잊는다. 특징에 의한 효과는 사라지지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【존재 상실】
+          문득 연기처럼 당신의 모습이 사라진다. 당신의 존재는 잠깐동안 이 세계에서 소실된다. 당신의 박탈치를 +1D6한다 (이 효과로는 징조표를 굴리지 않는다)
+        - |-
+          【이탈의 대가】
+          이탈능력을 사용한 결과로 징조표를 굴리게 되었을 경우에만 적용한다 (그렇지 않을 경우 다시 굴린다). 이 표를 굴리는 원인이 된 이탈능력을 이 라운드 (전투중이 아니라면 씬 동안) 동안 더 이상 사용할 수 없다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:288]: The trailing space should be removed.
```suggestion
          이탈능력을 사용한 결과로 징조표를 굴리게 되었을 경우에만 적용한다 (그렇지 않을 경우 다시 굴린다). 이 표를 굴리는 원인이 된 이탈능력을 이 라운드 (전투중이 아니라면 씬 동안) 동안 더 이상 사용할 수 없다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 13]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          태풍, 용오름, 눈보라, 폭염 등 이상기후가 발생한다. 배는 나뭇잎처럼 흔들리고 짐승은 공포에 허둥댄다. 이 씬 내에서 진행하는 모든 이동의 이동 거리가 반감한다.
+        - |-
+          【하늘의 분노】
+          일그러짐에 의해서 있을 수 없는 이상기후가 발생한다. 모든것을 날려보내는 바람이 불고 뇌명이 운다. 이 씬 중 모든 캐릭터는 비행상태가 될 수 없다.
+        - |-
+          【미궁의 숲】
+          주변의 풍경이나 지형이 지식이나 지도와 전혀 맞지 않는다. 어딘가로 향할 때 【지각】 난이도 12의 판정에 실패한 경우 헤멘 결과 【MP】 2D6점을 잃는다.
+        - |-
+          【죽음의 세계】
+          세계는 기묘하도록 아름답다. 물에는 물고기 한 마리 없고 덤불에는 벌레 한 마리 없다. 하늘에는 새가 날지 않는다. 기분나쁜 정적이 세계를 감싼다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:339]: The trailing space (tab) should be removed.
```suggestion
          세계는 기묘하도록 아름답다. 물에는 물고기 한 마리 없고 덤불에는 벌레 한 마리 없다. 하늘에는 새가 날지 않는다. 기분나쁜 정적이 세계를 감싼다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 14]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          연극의 무대가 된 지역 그 자차체가 허무에 삼켜져 사라진다. 영향하의 모든 캐릭터(가람을 포함)는 【재봉】 난이도 10의 판정에 성공하면 탈출할 수 있다. 실패한 경우 즉시 사망한다. 엑스트라는 무조건 사망한다.
+      DTS:
+        name: 일그러짐표(바다)
+        items:
+        - |-
+          【해양오염】
+          적조가 발생한다. 혹은 해수 그 자체가 독으로 변한다. 등장한 모든 캐릭터는 【체력】 난이도 10의 판정을 해 실패할 경우 [중독] 5를 받는다. 엑스트라가 어떻게 될지는 GM이 정하지만 치료하지 않는 한 얼마 안 가 사망할 것이다.
+        - |-
+          【색채침식】
+          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:372]: The trailing space should be removed.
```suggestion
          세계가 칠흑과 둔백으로 덧칠된다. 씬 내에 존재하는 여러 무생물과 생물이 흑백의 모노톤이 된다. 실잣는이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않아도 좋다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 15]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          해저화산이 분화한다. 해수는 끓어올라 주변의 생물들이 차례차례 사멸하고 바다지기들도 고통받는다. 바다의 물보라는 열탕이 된다. 구체적인 효과는 GM이 결정할 것.
+        - |-
+          【왜곡된 이정표】
+          일그러진 이정표가 내려온다. 일그러짐표（『MM』P263）를 굴릴 것.
+        - |-
+          【둔백의 바다】
+          바닷물이 둔백으로 변한다. 허무를 향한 달콤한 꼬득임으로 가득 찬 물에 닿은 자는 생각하는 것을 그만두고 의지 없는 인형이 되어 언젠가 녹아 사라질 것이다. 이 일그러짐이 1D6씬 (또는 1D6일) 이내에 사라지지 않을 경우 모든 캐릭터 (PC포함)는 엑스트라가 되어 두 번 다시 자아를 찾을 수 없다. GM이 어떠한 이유로 효과를 받지 않는 캐릭터를 설정해도 좋다.
+        - |-
+          【검은 바다】
+          바닷물이 칠흑으로 변한다. 바다 속에 있으면 한 치 앞도 보이지 않는다. [구분 : 사격]의 무기에 의한 공격의 펌블치가 +5된다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:387]: The trailing space (tab) should be removed.
```suggestion
          바닷물이 칠흑으로 변한다. 바다 속에 있으면 한 치 앞도 보이지 않는다. [구분 : 사격]의 무기에 의한 공격의 펌블치가 +5된다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 16]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+          해수가 민물이 된다. 인간 중에는 기뻐하는 자들도 있지만 바다의 생물들에게는 죽음을 의미한다.
+        - |-
+          【해류소실】
+          바다의 해류가 사라진다. 바다지기의 힘을 가지고도 파도를 일으킬 수 없다. 고인 해수는 부패하고 바다의 은혜를 받아 살아가는 이들이 차례차례 병에 걸린다.
+        - |-
+          【무너진 세계】
+          바다 주변 (바다 내부가 아니라 그 외부)에서 폭발적으로 올풀림이 넓어진다. 그 규모는 GM이 결정하지만 머지않아 주민들은 죽음을 맞이할 것이다.
+        - |-
+          【술의 바다】
+          바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다.
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:411]: The trailing space (tab) should be removed.
```suggestion
          바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다.
```

File: i18n/MonotoneMuseum/ko_kr.yml [hunk 17]
```diff
@@ -25,65 +25,538 @@ ko_kr:
...
+        items:
+        - |-
+          【폭주충동】
+          마음 속 깊은 곳에서부터 충동이 치솟는다. 눈 앞의 생명을 찢어버리고싶다ㅡ당신의 의지와 반대로 그 팔이 휘둘러진다. 지근거리에 있는 캐릭터 전원에게 즉시 <재봉> 2D6점의 대미지를 준다. 전투 중이 아닐 경우 어디까지를 지근으로 할지는 GM이 판단할것. 대미지를 줄 대상이 없는 경우 이 연극 중 당신의 행동치를 +1한다.
+        - |-
+          【균열】
+          격통이 일고, 당신의 몸에 금이 간다. 당신은 [2D6+배우 레벨]점의 【HP】를 잃는다.
+        - |-
+          【칠흑의 날개】
+          당신의 등에 오한과 가려움이 일고, 칠흑의 날개가 피어난다. 이 연극 중, 당신은 마이너 액션으로 박탈치 1을 상승시켜 비행상태가 된다. (해제에도 마이너 액션 사용)
```
Comment (ochaochaocha3) [inline i18n/MonotoneMuseum/ko_kr.yml:538]: The trailing space should be removed.
```suggestion
          당신의 등에 오한과 가려움이 일고, 칠흑의 날개가 피어난다. 이 연극 중, 당신은 마이너 액션으로 박탈치 1을 상승시켜 비행상태가 된다. (해제에도 마이너 액션 사용)
```

**Review by ochaochaocha3 — CHANGES_REQUESTED** [review]
Some tests have failed due to the syntax error in `test/data/MonotoneMuseum_Korean.toml` and trailing spaces in `i18n/MonotoneMuseum/ko_kr.yml`. Please confirm these files.

**Comment by jbblily** [comment]
Sorry for lots of errors... I deleted all the tabs at the end of the sentences, and also some typos not matching to test script. 
I hope it helps...

**Review by ochaochaocha3 — APPROVED** [review]
Your fixes seem good to me. Thank you for your cooperation!

**Comment by ysakasin** [comment]
@jbblily Awesome work! Thanks!

**Pull request merged**